packages/web-ui/.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+/** @type {import("eslint").Linter.Config} */
+module.exports = {
+  extends: ['./node_modules/@latitude-data/eslint-config/library.js'],
+  env: {
+    node: true,
+    browser: true,
+  },
+}
```

Comment:
This won't work on eslint 9. Steps to fix:
- rename it to .eslintrc.json (fix the config so that it's json also ofc)
- run npx @eslint/migrate-config .eslintrc.json
- run  globals @eslint/js @eslint/eslintrc to install the missing dependencies for eslint 9